src/Illuminate/Validation/Validator.php

```diff
@@ -242,14 +242,25 @@ protected function explodeRules($rules)
 
                 unset($rules[$key]);
             } else {
-                $rules[$key] = (is_string($rule)) ? explode('|', $rule) : $rule;
+                $rules[$key] = $this->extractRulesArray($rule);
             }
         }
 
         return $rules;
     }
 
     /**
+     * Extract the rules in an array form.
+     *
+     * @param  string|array $rules
+     * @return array
+     */
+    protected function extractRulesArray($rules)
+    {
+        return (is_string($rules)) ? explode('|', $rules) : $rules;
```

Comment:
The extra `()` for the condition are not required. You can do `is_string($rules) ? explode('|', $rules) : $rules;`